pages/modal.vue

```diff
@@ -0,0 +1,156 @@
+<template>
+  <div class="main-container">
+    <div class="first-section">
+      <div>
+        <svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
@SualaDev just so your code is cleaner, you can add your SVGs / Icons as components